src/bitcoind.cpp

```diff
@@ -55,10 +55,10 @@ static bool AppInit(int argc, char* argv[])
 
     // Process help and version before taking care about datadir
     if (HelpRequested(args) || args.IsArgSet("-version")) {
-        std::string strUsage = PACKAGE_NAME " version " + FormatFullVersion() + "\n";
+        std::string strUsage = PACKAGE_NAME " version " + FormatFullVersion() + "\n";//from init.cpp
 
         if (args.IsArgSet("-version")) {
-            strUsage += FormatParagraph(LicenseInfo()) + "\n";
+            strUsage += FormatParagraph(LicenseInfo()) + "\n" + CopyrightInfo() + "\n";//from init.cpp
```

Comment:
bitcoind.cpp uses the same strings that help and about use.
LicenseInfo() and CopyrightInfo() are now refactored for reusability.